salt/proxy/panos.py

```diff
@@ -355,13 +358,32 @@ def call(payload=None):
                                         decode_type='plain',
                                         decode=True,
                                         verify_ssl=False,
+                                        status=True,
                                         raise_error=True)
     except KeyError as err:
         raise salt.exceptions.CommandExecutionError("Did not receive a valid response from host.")
 
     if not r:
         raise salt.exceptions.CommandExecutionError("Did not receive a valid response from host.")
 
+    if str(r['status']) not in ['200', '201', '204']:
```

Comment:
Does this `str()` wrapping work on Py3? I think these might need to be `six.text_type` instead.

Reply:
I've updated this as requested.